src/components/Viewer/index.js

```diff
@@ -80,7 +80,7 @@ class Viewer extends Component {
   getLibraries = () => libraries
 
   getAssetURL = (filename) => {
-    return images[filename] || `${assetsBaseURL}/${filename}`
+    return `${assetsBaseURL}/${filename}`
```

Comment:
```suggestion
    return `(images && images[filename]) || ${assetsBaseURL}/${filename}`
```
this is less destructive in case images are used somewhere

Reply:
This is what I went for at first as well, but an error continued to be thrown because `images` is not defined in the file anywhere.